README.md

```diff
@@ -1,7 +1,13 @@
 This repository was created to post javascript files and open issues related to them, fix bugs in codes, write functions and other activities associated with programming in javascript, all while building skills and confidence with GitHub.
 
+Here is how the workflow works in this repository:
 
-Your first step is to "fork" this repository into your own GitHub environment. See the guide below for assistance.
+ - For each new challenge, the author creates a new branch named after the task and creates an issue describing the challenge.
```

Comment:
I changed in your text the name of the branch from function to task, to generalize it, so just in case the challenge is not a function.